src/host/screenInfo.cpp

```diff
@@ -1016,23 +1001,7 @@ void SCREEN_INFORMATION::ProcessResizeWindow(const til::rect* const prcClientNew
     // Only attempt to modify the buffer if something changed. Expensive operation.
     if (coordBufferSizeOld != coordBufferSizeNew)
     {
-        auto& commandLine = CommandLine::Instance();
-
-        // TODO: Deleting and redrawing the command line during resizing can cause flickering. See: http://osgvsowi/658439
-        // 1. Delete input string if necessary (see menu.c)
-        commandLine.Hide(FALSE);
-
-        const auto savedCursorVisibility = _textBuffer->GetCursor().IsVisible();
-        _textBuffer->GetCursor().SetIsVisible(false);
-
-        // 2. Call the resize screen buffer method (expensive) to redimension the backing buffer (and reflow)
         LOG_IF_FAILED(ResizeScreenBuffer(coordBufferSizeNew, FALSE));
-
-        // MSFT:19976291 Don't re-show the commandline here. We need to wait for
-        //      the viewport to also get resized before we can re-show the commandline.
-        //      ProcessResizeWindow will call commandline.Show() for us.
-        _textBuffer->GetCursor().SetIsVisible(savedCursorVisibility);

```

Comment:
The reason the old code has these `commandLine.Hide();` and `Show()` calls everywhere (at least 5 different places) is because of this one exception here where it explicitly doesn't `Show()` it again immediately.

But I read into MSFT:19976291 (summary: `COOKED_READ_DATA` could crash conhost when you resize the window quickly & randomly) and then tried to test it with my reimplementation and I don't think this issue can happen anymore, because my `EraseBeforeResize()` and `RedrawAfterResize()` is more robust.

As such all of these are gone and moved a couple lines below in this diff into `SCREEN_INFORMATION::ResizeScreenBuffer`.